gapic/samplegen/samplegen.py

```diff
@@ -58,7 +59,28 @@
 TEMPLATE_NAME = "sample.py.j2"
 
 
-TransformedRequest = namedtuple("TransformedRequest", ["base", "body"])
+@dataclasses.dataclass(frozen=True)
+class AttributeRequestSetup:
+    value: str
+    field: Optional[str] = None
+    value_is_file: bool = False
+    input_parameter: Optional[str] = None
+    comment: Optional[str] = None
```

Comment:
Empty string is probably better than None for these last two.

Reply:
The only keys the request input dict is required to have are "field" and "value"; the others are optional. I feel that `Optional[str]` is a better representation, but if you disagree I will change to an empty str default.